### ISSUE TITLE
relref compatible references

### ISSUE THREAD

**Issue by opoudjis** [riboseinc/asciidoctor-bibliography#56]
In asciidoctor-rfc, I have special processing of particular replacement text in cross references:

````
<<RFC5322,2.1.1 comma>>
````

is expected to convert into

````
<relref section="2.1.1" displayFormat="comma" target="RFC5322"/> (v3)
<xref target="RFC5322">[RFC5322], Section 2.1.1</xref> (v2)
````

Whatever is going on in cite:[xxxx] with asciidoctor-bibliography, I want those cross-references to work like that still. Right now they don't, because the target `RFC5322` is being replaced by asciidoctor-bibliography with `bibliography-RFC5322`.

Even if that didn't happen, I would expect that `cite` macros would permit me my own alternate text, just as `<<crossref,alttext>>` does in Asciidoctor, and `<xref target="crossref">alttext</xref>` does in RFC XML. The fact that asciidoctor-bibliography supplies its own crossreference text depending on style is irrelevant to the requirements of RFC XML.

So in fact, I would expect `cite:[xxxx]` and `cite:[xxxx,yyyy]` to behave identically to `<<xxxx>>` and `<<xxxx,yyyy>>` in Asciidoctor, and to be processed by `inline_anchor_xref()` in asciidoctor-rfc.

I know these tickets are likely a radical change in spec to asciidoctor-bibliography. But without them, this gem does not contribute to authoring usable RFC XML.

**Comment by paolobrasolin** [comment]
That's a fair point. The focus was on using CSL (and TeX)  styles with ease, so no "arbitrary text" citation macro exists. Definitely a must-have, though. (Same goes for the arbitrary link taget, at least to ease inter-operation with other extensions.)

**Comment by ronaldtse** [comment]
@opoudjis @paolobrasolin I actually didn't understand the issue before but now I finally get it. This is indeed a must-have!

**Comment by ronaldtse** [comment]
I just tried today and unfortunately it doesn't work yet. Any ETA @paolobrasolin ? Thanks!

**Comment by paolobrasolin** [comment]
Will get this done before the week ends.

**Comment by ronaldtse** [comment]
Appreciate it @paolobrasolin !

**Comment by paolobrasolin** [comment]
This issue will be closed along with #54, since the two are strictly related; I'm splitting the feature request (non strictly necessary for closure) into #59.
